platform/android/src/style/sources/source.cpp

```diff
@@ -43,6 +43,10 @@ namespace android {
         return jni::Make<jni::String>(env, source.getID());
     }
 
+    jni::String Source::getAttribution(jni::JNIEnv& env) {
+        return source.getAttribution() ? jni::Make<jni::String>(env, source.getAttribution().value()) : jni::Make<jni::String>(env,"");
```

Comment:
done in https://github.com/mapbox/mapbox-gl-native/pull/8630/commits/527ffbe733979f393dafd23a062d895e524ea294#diff-8a5300c6fbc51b88f2e971461adcb513R33. re-review that part of the code?